olp-cpp-sdk-dataservice-write/tests/StreamLayerClientImplTest.cpp

```diff
@@ -41,6 +73,12 @@ class MockStreamLayerClientImpl : public StreamLayerClientImpl {
               (model::PublishSdiiRequest request,
                client::CancellationContext context),
               (override));
+
+  PublishDataResponse TestPublishDataLessThanTwentyMibSync(
+      const model::PublishDataRequest& request,
+      client::CancellationContext context) {
+    return PublishDataLessThanTwentyMibSync(request, context);
+  }
```

Comment:
can be replaced with using StreamLayerClientImpl::PublishDataLessThanTwentyMibSync